src/list-pulls.js

```diff
@@ -23,7 +23,9 @@ var apputil = require('./apputil');
 var flagutil = require('./flagutil');
 var repoutil = require('./apputil');
 
-var GITHUB_API_URL = "https://api.github.com/";
+// Set env variable CORDOVA_GIT_ACCOUNT to <username>:<apitoken> to avoid hitting GitHub rate limits.
```

Comment:
my understanding was that it's just <apitoken> not <username>:<apitoken>
eg. 
curl -s -I -H Authorization: token <apitoken> https://api.github.com/users/apache/repos